### PR TITLE
#135 fix province typo and export api update

### PR DESCRIPTION
**Pull request by navins94**

### CODE REVIEW

**Comment by wayangalihpratama** [comment]
Hi @navins94 ,
This PR ready for review ?

**Comment by navins94** [comment]
> Hi @navins94 , Is this PR ready for review?

It is ready now

**Review by wayangalihpratama — APPROVED** [review]

**Comment by wayangalihpratama** [comment]
Hi @navins94 , 
I made some updates to this branch. Please pull first.
I try to export data from the map page without any filter, the API fetch sends all data_ids as param. Can you fix that, please?
By default when no filtering is defined, we shouldn't send data ids param.

File: frontend/src/pages/dashboard/Maps.jsx [hunk 1]
```diff
@@ -31,6 +33,26 @@ function Maps() {
     });
   };
 
+  const handleExport = () => {
+    setExportLoading(true);
+    const dataIdsFilter = mapData
+      .flatMap((x) => x.id)
+      .map((x) => encodeURIComponent(x))
+      .join("&data_ids=");
+    const url = `download/data?data_ids=${dataIdsFilter}`;
```
Comment (wayangalihpratama) [inline frontend/src/pages/dashboard/Maps.jsx:42]: We shouldn't send data ids param when the user does not filter the data.

**Review by wayangalihpratama — CHANGES_REQUESTED** [review]

**Pull request merged**